src/js/views/hallOfFame.js

```diff
@@ -42,16 +43,32 @@ async function updatePlayers(inputs, updateEvents) {
                 }
             }
 
-            players[i].bestStats = {
-                gp: 0,
-                min: 0,
-                per: 0,
-            };
+            players[i].bestStats = {};
+            let bestEWA = 0;
+            players[i].legacy = {};
             for (let j = 0; j < players[i].stats.length; j++) {
-                if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
+                const rEWA = players[i].stats[j].ewa;
+                const team = players[i].stats[j].abbrev;
+                let pEWA = 0;
+                for (let k = 0; k < players[i].statsPlayoffs.length; k++) {
+                    if (players[i].stats[j].season === players[i].statsPlayoffs[k].season && team === players[i].statsPlayoffs[k].abbrev) {
+                        pEWA = players[i].statsPlayoffs[k].ewa;
+                        break;
+                    }
+                }
+                if (rEWA + pEWA > bestEWA) {
+                    bestEWA = rEWA + pEWA;
                     players[i].bestStats = players[i].stats[j];
```

Comment:
I guess I'm arguing that a player who actually makes it to the playoffs (as opposed to one who doesn't) is therefore more statistically dominant, because their contribution *mattered* and wasn't wasted on an underperforming team.

Converse: a player who is injured for >10 games digs themselves a nearly unescapable hole because they have fewer games to add up stats. It would hurt their ranking just as much as >10 games of playoffs would help them. If we don't count playoffs, shouldn't we also normalize injuries?